src/main/java/com/sonyericsson/hudson/plugins/gerrit/trigger/NamedGerritEventListener.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 CloudBees Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.sonyericsson.hudson.plugins.gerrit.trigger;
+
+import com.sonymobile.tools.gerrit.gerritevents.GerritEventListener;
+
+/**
+ * A {@link GerritEventListener} that can provide a display name.
+ * Nice to have when displaying this on {@link com.sonyericsson.hudson.plugins.gerrit.trigger.diagnostics.Diagnostics}.
+ *
+ * @author Robert Sandell &lt;rsandell@cloudbees.com&gt;.
+ */
+public interface NamedGerritEventListener extends GerritEventListener {
+
+    /**
+     * A possible more descriptive display name than the class name.
+     *
+     * @return the name
+     */
+    String getDisplayName(); //TODO Replace with a default method in GerritEventListener when moving to Java 8
```

Comment:
Is this allowed to return null?  This one would be very helpful to add the appropriate Findbugs annotation to.